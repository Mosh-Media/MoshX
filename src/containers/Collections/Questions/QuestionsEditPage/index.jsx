import React from "react";

const QuestionsEditPage = () => {
  return <h1>Todo</h1>;
};

export default QuestionsEditPage;
