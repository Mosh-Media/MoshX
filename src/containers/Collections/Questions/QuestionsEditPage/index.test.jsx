import React from "react";

import ShallowRenderer from "react-test-renderer/shallow";
import QuestionsEditPage from ".";

const renderer = new ShallowRenderer();

describe("QuestionsEditPage Component", () => {
  test("should match the snapshot", () => {
    renderer.render(<QuestionsEditPage />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
