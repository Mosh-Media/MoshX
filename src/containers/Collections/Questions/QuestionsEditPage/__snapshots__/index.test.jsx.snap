// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionsEditPage Component should match the snapshot 1`] = `
<ForwardRef(WithStyles)
  maxWidth="lg"
>
  <ForwardRef(WithStyles) />
  <h1>
    Todo
  </h1>
</ForwardRef(WithStyles)>
`;
